docs/src/getting-started/running-scripts.md

```diff
@@ -22,3 +22,16 @@ Script calls provide the same logging functions, `get_logs()` and `get_logs_with
 {{#include ../../../packages/fuels/tests/logs.rs:script_logs}}
 ```
 
+## Calling contracts from scripts
+Scripts can use the same interfaces for setting external contracts as [contract methods](../calling-contracts/other-contracts.md).
+
+As a reminder, `set_contracts(&[&contract_instance, ...])` requires contract instances that were created using the `abigen` macro. When setting the external contracts with this method, logs and require revert errors originating from the external contract can be propagated and decoded by the calling contract.
```

Comment:
```suggestion
```

Reply:
^ I would say remove this because you already linked to the correct doc + provided a code snippet. DRY.